integration_test/buckets.tf.template

```diff
@@ -30,17 +30,6 @@ resource "aws_s3_bucket" "level0" {
     Application = "ORCA"
   }
 }
-resource "aws_s3_bucket_lifecycle_configuration" "level0-lifecycle-rule" {
-  bucket = aws_s3_bucket.level0.id
-  rule {
-    id = "move all objects to glacier"
-    status = "Enabled"
-    transition {
-      days          = 0
-      storage_class = "GLACIER"
-    }
-  }
-}
```

Comment:
Do we want this? Doesn't seem to map to any ORCA thing.

Reply:
I don't believe so unless we need it for testing. I was just looking at the rest, this may actually be for a bucket test case where we are restoring data to a Glacier bucket in Cumulus. So we may want to keep it for that. In essence LP and a few others archive level 0 data but that data is not distributed to people generally so they can store it in a Glacier/Deep_Archive storage class.

Reply:
Alright. I wouldn't think a lifecycle rule could stop a file from being added to a bucket, but I've seen weirder. Replaced.